### ISSUE TITLE
Write roadmap context for website

### ISSUE THREAD

**Issue by andyxerxes** [lsst-uk/lasair#70]
Web page setting Lasair-ZTF in context of Lasair-LSST development

**Comment by smarttgit** [comment]
On the front page of lasair website - add a sidebar link "Lasair-LSST"

While we have released Lasair 2.0 as a prototype broker based on the ZTF alert stream, we have started an intensive planning and technical reviews of the requirements for LSST.  We are basing our future requirements on two key LSST documents. One is the Data Products Definition Document (Juric et al. 2019; https://lse-163.lsst.io/) and LSST Alerts: Key Numbers (Graham et al. 
https://dmtn-102.lsst.io/DMTN-102.pdf). 

We envisage  the following major differences between Lasair-ZTF and Lasair-LSST  : 

-  the increase in rate of the number of alerts, by a factor 50 
-  the multi-colour filter nature of LSST of ugrizy, 
-  the different surveys : deep drilling fields, wide-fast deep (which may end up splitting the Milky Way from the extragalactic sky), and any additional approved mini-surveys (e.g. target of opportunity observations of gravitational wave sky maps). 
-  access to forced photometry and the timescales for that access 
- a deep image of the whole survey footprint which has photometric redshifts and accurate star-galaxy separation for all objects 
- expected large increase in the user base 
- provision to link pubic spectroscopic data to the LSST transients. Both the information derived from the spectra such as type and redshift, but also the data itself. 

Given these differences, within Lasair we are reviewing the database architecture and how the lightcurve data are annotated, stored and made available to users. We have developed detailed Science Requirements from the perspective of expert users within the UK and are working particularly with the LSST Science Collaborations (DESC and TVS) to review these. Our main focus is to provide a broker that will allow users to find their transients and eruptive variables of choice and to run filters or analysis code on them. This may be on the Lasair computing platform (the UK's IRIS system) but we will also provide means to bulk download lightcurve data for users to work with locally on their own machines. 

This work is intensifying as of April 2020, and we look forward to working with our international colleagues across the LSST community and within the LSST Science Collaborations to define Lasair's functionality for LSST. With this in mind, we are drafting our broker proposal for the June call.
